Makefile

```diff
@@ -1,31 +1,38 @@
-TARGETS = naive cuda cupla-cuda-async cupla-seq-seq-async cupla-seq-seq-sync cupla-tbb-seq-async cupla-omp2-seq-async kokkos-serial kokkos-openmp kokkos-cuda
+TARGETS = naive alpakaSer alpakaTBB alpakaGpuCuda cuda cupla-cuda-async cupla-seq-seq-async cupla-seq-seq-sync cupla-tbb-seq-async cupla-omp2-seq-async kokkos-serial kokkos-openmp kokkos-cuda
 
 .PHONY: all debug clean $(TARGETS)
 
-CXX := g++-8
-CXX_FLAGS := -O2 -std=c++14 -ftemplate-depth-512
+EIGEN_ROOT := /data/cmssw/slc7_amd64_gcc820/external/eigen/e4c107b451c52c9ab2d7b7fa4194ee35332916ec-pafccj/include/eigen3
+TBB_ROOT := /data/cmssw/slc7_amd64_gcc820/external/tbb/2019_U3-pafccj
+BOOST :=  
+
+CXX := g++ 
+CXX_FLAGS := -O2 -std=c++14 -ftemplate-depth-512 -I/data/cmssw/slc7_amd64_gcc820/external/boost/1.67.0-pafccj/include/ -I$(TBB_ROOT)/include -L$(TBB_ROOT)/lib
 CXX_DEBUG := -g
 
-NVCC := /usr/local/cuda-10.1/bin/nvcc -ccbin $(CXX)
-NVCC_FLAGS := -O2 -std=c++14 --expt-relaxed-constexpr -w --generate-code arch=compute_50,code=sm_50
+
+
+NVCC := /data/cmssw/slc7_amd64_gcc820/external/cuda/10.1.105-pafccj2/bin/nvcc -ccbin $(CXX)
+NVCC_FLAGS := -O2 -std=c++14 --expt-relaxed-constexpr -w -lineinfo -DALPAKA_CUDA_ARCH=60:70:75 -I/data/cmssw/slc7_amd64_gcc820/external/boost/1.67.0-pafccj/include/ -I$(TBB_ROOT) -L$(TBB_ROOT)/lib
```

Comment:
I think it would be cleaner if `-DALPAKA_CUDA_ARCH=60:70:75` would be added only for Alpaka targets.

Also, why does `nvcc` need to know about TBB?